asset/css/site-nav.css

```diff
@@ -140,7 +143,7 @@
 
 /* Responsive site navigation */
 
-@media screen and (max-width: 767px) {
+@media screen and (max-width: 1006px) {
```

Comment:
I am fine with `1006px`, although personally I think it doesn't look like a nice number compared to like `999px`. :P

Reply:
The old media query had the `#page-content` div squished to 416 pixels width when the window width was 786 pixels. This was too narrow to read and triggered a much more dramatic transition effect.

So I took the maximum width of Panel in responsive mode (window at 767 pixels or lower), which was 655 pixels and subtracted the squished width of 400+ pixels. The result was 239 pixels.

239 + 767 = 1006 😃. So now the `#page-content` div will have a minimum of 655 pixels of width  when the window width is above 767 pixels.

Reply:
Hmm, that should be in a code comment. In general, stick to Bootstrap's [responsive breakpoints](https://getbootstrap.com/docs/4.0/layout/overview/#responsive-breakpoints).